### ISSUE TITLE
LAST24/NEXT24 BMT

### ISSUE THREAD

**Issue by sgoEma** [sgoEma/EventManagement#60]

**Comment by bmtemakina** [comment]
Hi @secoEmakina , @sgoEma , @sercanEmakina 
#Last 24 Hours:

- I began to search solutions for #37 and wrote two solution analyses.
- I finished the Battle Station App Project badge on the trailhead.

#Next 24 Hours:

- I will be searching for new solutions on #37 and if any of them gets approved I will be implementing the solution.
- For today I will be working on Lightning App Builder Module on Trailhead

Best,

**Comment by bmtemakina** [comment]
Hi @sgoEma , @secoEmakina , @sercanEmakina ,

#Last 24 Hours:

- I started to implement chosen solution for #37 . I created the after insert flow and added an scheduled path to it for sending emails.
- I tested the flow a little.
- I finished the Lightning App Builder Module on Trailhead.

#Next 24 Hours:

- I'm going to test the flow that I created for #37 , find a solution for date updates, and open a pull request for the task.
- ELD
- After #37 I'm going to take another task for myself.
- I'm going to be starting to App Customization Specialist SuperBadge on Trailhead.

Best,

**Comment by bmtemakina** [comment]
Hi @sgoEma  , @sercanEmakina , @secoEmakina ,

#Last 24 Hours:

- I came across with a little problem on #37 but solved it and just waiting for solution approval.
- I tested the flow with multiple cases.
- I created a draft Pull Request with test cases.
- ELD
- I started to App Customization Specialist SuperBadge on Trailhead.

#Next 24 Hours:

- If new update on solution is acceptable, I will open the Pull Request.
- I'm going to make solution analysis for the new Task.
- I'm going to finish the App Customization Specialist SuperBadge on Trailhead.

Best,

**Comment by bmtemakina** [comment]
Hi @sgoEma , @sercanEmakina , @secoEmakina ,

#Last 24 Hours:

- I finished #37 task and merged it after PR got approved.
- I wrote a solution for #55 and implemented it
- I finished App Customization Specialist SuperBadge on Trailhead.

#Next 24 Hours:

- I'm going to open a Pull Request after few test for #55 .
- I'm going to make a solution analysis for the new Task and then implement the chosen one.
- I'm going to be starting to the Change Set Development Model Badge on Trailhead
- ELD

Best,

**Comment by bmtemakina** [comment]
Hi @sgoEma , @sercanEmakina , @secoEmakina ,

#Last 24 Hours:

- I finished #65 and created a draft Pull Request.
- ELD
- I finished Change Set Development Model Badge on Trailhead.
- I updated the test cases for #37 .
- I reviewed the changes on #64 

#Next 24 Hours:

- I'm going to merge the changes if PR gets approved.
- I'm going make solution analysis for the new task.
- ELD
- I'm going to be starting to the Org Development Model module on Trailhead.

Best,

**Comment by bmtemakina** [comment]
Hi @sgoEma , @sercanEmakina , @secoEmakina ,

#Last 24 Hours:

- I opened a PR.
- I made a little research for #49 solutions.
- I finished the Org Development Model module on Trailhead.
- ELD

#Next 24 Hours:

- I'm going to write solution for #49 and then will implement the solution.
- I'm going to finish the Data Modeling and App Deployment module and then look up example questions for the App Builder exam.

Best,

**Comment by bmtemakina** [comment]
Hi @sgoEma , @sercanEmakina , @secoEmakina ,

#Last 24 Hours:

- I wrote a solution for #49 .
- I finished the Data Modeling and App Deployment module on Trailhead
- I Looked up some example questions for App Builder exam

#Next 24 Hours:

- If solution for #49 gets approved, I will start to implement it.
- I'm waiting for PR approval
- ELD
- I'm going to study for App Builder exam.

Best,

**Comment by bmtemakina** [comment]
Hi @sgoEma , @sercanEmakina , @secoEmakina ,

#Last 24 Hours:

- I merged my PR.
- I studied example questions for App Builder exam.
- ELD
- Some preparation for Sprint Demo 3

#Next 24 Hours:

- I'm going to prepare for Sprint Demo 3
- I'm going to study for App Builder exam.
- I'm going to study some modules on Trailhead again.

Best,

**Comment by bmtemakina** [comment]
Hi @sgoEma , @sercanEmakina , @secoEmakina ,

#Last 24 Hours:

- We presented the Sprint Demo 3.
- I looked up some modules on Trailhead.
- I examined questions for App Builder.

#Next 24 Hours:

- I will continue to study for App Builder.
- ELD
- I may will continue to tasks for Event Management.

Best,

**Comment by bmtemakina** [comment]
Hi @sgoEma , @sercanEmakina , @secoEmakina ,

#Last 24 Hours:

- I continued to study for App Builder
- ELD

#Next 24 Hours:

- I'll examine some tasks on Event Management
- I'll continue to study for App Builder
- I'll examine the POC
- ELD

Best,

**Comment by bmtemakina** [comment]
Hi @sgoEma , @sercanEmakina , @secoEmakina ,

#Last 24 Hours:

- Examined the POC
- Examined example App Builder questions

#Next 24 Hours:

- Work on #49 
- Examine POC
- Study for App Builder

Best,

**Comment by bmtemakina** [comment]
Hi @sgoEma , @sercanEmakina , @secoEmakina ,

#Last 24 Hours:

- Worked on #49 
- Studied example App Builder questions

#Next 24 Hours:

- Continue to work on #49 
- Study for App Builder

Best,

**Comment by bmtemakina** [comment]
Hi @sgoEma , @sercanEmakina , @secoEmakina ,

#Last 24 Hours:

- Finished #49 .
- Studied example App Builder questions.

#Next 24 Hours:

- Test the #49 solution and write Test Cases.
- Continue to study for App Builder.

Best,

**Comment by bmtemakina** [comment]
Hi @sgoEma , @sercanEmakina , @secoEmakina ,

#Last 24 Hours:

- Tested the #49 and fixed some bugs
- Studied for App Builder

#Next 24 Hours:

- Work on test cases for #49 
- App Builder study

Best,

**Comment by bmtemakina** [comment]
Hi @sgoEma , @sercanEmakina , @secoEmakina ,

#Last 24 Hours:

- Tested the #49 and fixed some bugs.
- Worked on test cases.

#Next 24 Hours:

- Finish test cases.
- Study for App Builder

Best,

**Comment by bmtemakina** [comment]
Hi @sgoEma , @sercanEmakina , @secoEmakina ,

#Last 24 Hours:

- Wrote test cases for #49.
- Studied for App Builder.

#Next 24 Hours:

- Open a PR for #49.
- Continue with #54.
- Continue to study for App Builder.

Best,

**Comment by bmtemakina** [comment]
Hi @sgoEma , @sercanEmakina , @secoEmakina , @alpertungagulbahar ,

#Last 24 Hours:

- I opened the  #69 PR.
- I started to the #58 task.
- I examined App Builder Questions.

#Next 24 Hours:

- I will merge #49 
- I will continue to work on #58 task.
- I will continue to study for App Builder

Best,

**Comment by bmtemakina** [comment]
Hi @sgoEma , @sercanEmakina , @secoEmakina , @alpertungagulbahar ,

#Last 24 Hours:

- I merged the #69 .
- I continued to work on #58 task.
- I studied for App Builder exam.

#Next 24 Hours:

- I will finish the #58 task and start to the next task.
- I will examine the App Builder exam questions.

Best,

**Comment by bmtemakina** [comment]
Hi @sgoEma , @sercanEmakina , @secoEmakina , @alpertungagulbahar ,

#Last 24 Hours:

- I made a research about flow test classes.
- Studied for App Builder.

#Next 24 Hours:

- Waiting for #70 because I think this task blocks the other tasks.
- Continue to study for App Builder.

Best,

**Comment by bmtemakina** [comment]
Hi @sgoEma , @sercanEmakina , @secoEmakina , @alpertungagulbahar ,

#Last 24 Hours:

- Worked on #54 .
- Wrote a test class for Before Insert and Update Session Flow.

#Next 24 Hours:

- Finish #54 and open a PR.
- continue with the next task.

Best,

**Comment by bmtemakina** [comment]
Hi @sgoEma , @sercanEmakina , @secoEmakina , @alpertungagulbahar ,

#Last 24 Hours:

- Sprint 5 demo
- research for #50 

#Next 24 Hours:

- Merge #83 .
- I will work on #50 

Best,

**Comment by bmtemakina** [comment]
Hi @sgoEma , @sercanEmakina , @secoEmakina , @alpertungagulbahar ,

#Last 24 Hours:

- Request for Address Finder app installation.
- App Builder study.
- Merged #83 .
- health maintenance :)

#Next 24 Hours:

- I will work on #68 .
- Salesforce global kick-off event.

Best,
